src/tools/dart-fix.md

```diff
@@ -0,0 +1,31 @@
+---
+title: The dart fix command
+description: Help for bringing your code up-to-date.
+toc: false
+---
+
+The `dart fix` command finds and fixes two types of issues:
+
+* Analysis issues that have associated automated fixes
```

Comment:
We also refer to the automated fixes as 'quick fixes' - I don't know if we want to mention that name here or not.

Reply:
Yes, I think it's good to mention synonyms that people might look for, for searchability/understability reasons. Thanks!

Reply:
The term 'quick-fix' was is used by IntelliJ and Eclipse. VS Code (and hence LSP) uses the term 'code action'. We should probably include both to cover a wider range of users.